lib/pronto/version.rb

```diff
@@ -1,6 +1,6 @@
 module Pronto
   module Version
-    STRING = '0.7.1'.freeze
+    STRING = '0.7.2'.freeze
```

Comment:
eager to lift the version to `0.8.0` but this requires pronto-rubocop gem update :/